### PR TITLE
warsow,warsow-engine: fix mirror URL

### PR DESCRIPTION
**Pull request by delroth**
###### Motivation for this change

Current one does not resolve. New one is the one currently present on warsow.net. Hashes unchanged.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @astsmtl @abbradar

### CODE REVIEW

**Comment by mmahut** [comment]
On ```aarch64-linux```:

```
[  9%] Building C object cgame/CMakeFiles/cgame.dir/__/gameshared/gs_players.c.o
g++: error: unrecognized command line option '-msse2'
```

File: pkgs/games/warsow/engine.nix [hunk 1]
```diff
@@ -24,6 +24,10 @@ in stdenv.mkDerivation (libs // rec {
     libpng
   ];
 
+  prePatch = stdenv.lib.optional stdenv.isAarch64 ''
```
Comment (multun) [inline pkgs/games/warsow/engine.nix]: how about `prePatch = stdenv.lib.optional !stdenv.isx86` ?

Comment (mmahut) [reply pkgs/games/warsow/engine.nix]: @multun could be, not sure about other targets such as darwin. But it doesn't build on ```aarch64``` anyway. So maybe we should exclude it, I'm pretty sure the game won't run on arm anyway.

Comment (mmahut) [reply pkgs/games/warsow/engine.nix]: Marked as broken on aarch64.

**Pull request merged**